lib/core/CustomRequestHandler.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const logger = require('../util/loggerFactory');
+
+/*
+This class handles the custom requests made to existing playwright connection.
+This is kept as Singleton class because we want to maintain a map of commands for which
+we can resolve the response once we receive from Playwright server.
+Please don't change the singleton behaviour of this class.
+*/
+class CustomRequestHandler {
```

Comment:
Can we please use [JSDOC](https://jsdoc.app/tags-example) comments for documenting the methods? For reference, you may refer to other files in the same folder.